src/sass/_nav.scss

```diff
@@ -8,7 +8,6 @@
   left:250px;
   background:$white;
   padding:5px;
-  height:41px;
```

Comment:
I dropped the height in efforts to allow for multiple menu-bars to be possibly added within the `nav#menu-bar` element (this is probably not likely).  I figured it would also help for lower-resolution screens (rather than pushing menus out of view).

Without the `height` styling, the menu bar appears as it did before.